<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>104</width>
      <height>74</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_copy">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;&gt;&gt;&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>292</width>
      <height>197</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QPushButton" name="pushButtonStart">
         <property name="text">
          <string>START GAME</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Clique nos botoes conforme aparece no visor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLCDNumber" name="lcdNumber_visor">
         <property name="segmentStyle">
          <enum>QLCDNumber::Flat</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>SequÃªncia atual:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumber_atual">
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>T. Reacao (ds):</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumber_maior">
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                 <property name="value" stdset="0">
                  <double>0.000000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButtonDez">
           <property name="text">
            <string>DEZ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonVinte">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>VINTE</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonTrinta">
           <property name="text">
            <string>TRINTA</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_copy</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDez</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>contador_azul()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>318</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonVinte</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>contador_azul()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDez</sender>
   <signal>clicked(bool)</signal>
   <receiver>lcdNumber_atual</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>copy()</slot>
  <slot>contador_azul()</slot>
  <slot>lcd_display(int)</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
